zeroeq/detail/receiver.h

```diff
@@ -11,15 +11,18 @@
 
 #include "../log.h"
 
+#include <servus/listener.h>
 #include <servus/servus.h>
 #include <zmq.h>
 
+#include <algorithm>
+
 namespace zeroeq
 {
 namespace detail
 {
 /** Manages and updates a set of connections with a zeroconf browser. */
-class Receiver
+class Receiver : public servus::Listener
```

Comment:
I have the impression that this class has the wrong name. Is it not a ConnectionManager instead of a Receiver?